keyboards/ziz0gaming19/c13x/keyboard.json

```diff
@@ -0,0 +1,65 @@
+{
+    "manufacturer": "Milos Lalic",
+    "keyboard_name": "c13x",
+    "maintainer": "miloslalic19",
+    "development_board": "promicro",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "matrix_pins": {
+        "cols": ["F7", "B1", "B3", "B2", "B6", "C6", "D7", "E6", "B4", "B5"],
+        "rows": ["F6", "F5", "D0", "D4"]
+    },
+    "processor": "atmega32u4",
+    "url": "https://github.com/ziz0gaming/c13x",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0002",
+        "vid": "0x4D4C"
+    },
+    "layouts": {
+        "LAYOUT": {
```

Comment:
The representation here should be closer to the physical layout

<img width="475" height="221" alt="image" src="https://github.com/user-attachments/assets/1ec16602-61d0-4563-86fa-5a929b9c4517" />